File: public/phase-2-20160620022203/PYTHON/flowers-girls-and-grandma.py
#!/usr/bin/python
# Name : Flowers Girls and Grandma
# input:
# n :  the number of girls
# p : Grandmas favourite prime number
# lr : each row corresponds to the range of flowers that each girl can grow
elements = raw_input().strip().split(' ')
n = int(elements[0])
p = int(elements[1])
lr = []
for _ in xrange(n):
	lr_elements = raw_input().strip().split(' ')
	lr.append(map(int, lr_elements))


# write your code here
# store your results in `result`

# output

# Dummy Data
result = 0.0

print(result)

File: public/phase-2-20160620022203/C/flowers-girls-and-grandma.c
// Name : Flowers Girls and Grandma
#include <stdio.h>
int main() {
	// input:
	// n :  the number of girls
	// p : Grandmas favourite prime number
	// lr : each row corresponds to the range of flowers that each girl can grow
	int n, p, lr[10000][2], idx, jdx;
	float result=0.0;
	char lr_tmp[2];
	scanf("%d %d", &n, &p);
	for(idx = 0; idx< n; idx++) {
		scanf("%s", &lr_tmp[0]);
		for(jdx = 0;jdx<2;jdx++) {
			lr[idx][jdx] = lr_tmp[jdx];
		}
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%f", result);
	return 0;
}